### ISSUE TITLE
Problem creating TFRecord files

### ISSUE THREAD

**Issue by agaldran** [mikevoets/jama16-retina-replication#2]
Hi there,

First, thanks for this awesome resource. I am starting to try to reproduce your experiments on my computer (OS: ubuntu 16.04), and I'm having some trouble with the first step of the "Preprocessing before training" section on the readme. 

Specifically, when I execute the command `git submodule update --init` I get the following on the terminal:

```
Cloning into 'create_tfrecords'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:mikevoets/create_tfrecords.git' into submodule path 'create_tfrecords' failed
```

Could you give me a hint about why could this be happening? I have limited knowledge on git, unfortunately..

Thanks!

Adrian

**Comment by mikevoets** [comment]
Hi Adrian! Can you pull the latest changes and try again? I changed the git submodule path to https, removing the requirement of having a Github account configured with a ssh key.

**Comment by agaldran** [comment]
Hi Mike,

It works perfectly now:

```
Submodule 'create_tfrecords' (https://github.com/mikevoets/create_tfrecords.git) registered for path 'create_tfrecords'
Cloning into 'create_tfrecords'...
remote: Counting objects: 86, done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 86 (delta 2), reused 4 (delta 1), pack-reused 77
Unpacking objects: 100% (86/86), done.
Checking connectivity... done.
Submodule path 'create_tfrecords': checked out '83d38816b64d35d217f8eff9ef98387fcb8af0bb'
```

I will proceed through the rest of the readme instructions and let you know here if I find any other problem. 

Thanks!

**Comment by mrzhangzizhen123** [comment]
@agaldran I recently trying to reappear the  code, I met a problem, but before I kaggle downloaded DR data set, contains five seven training set and test set, but I put these data sets in eyepacs folder after running eyepacs. Sh extract only the training set, after running eyepacs. Sh is only a pool folder, inside a folder called "train" and a lot of pictures, but the train is empty inside.I only see one pool file when I run eyepacs.sh. Can you help me solve it?
